saos-import/src/main/java/pl/edu/icm/saos/importer/commoncourt/judgment/remove/SourceCcRemovedJudgmentsFinder.java

```diff
@@ -0,0 +1,83 @@
+package pl.edu.icm.saos.importer.commoncourt.judgment.remove;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import pl.edu.icm.saos.importer.commoncourt.judgment.download.SourceCcjExternalRepository;
+import pl.edu.icm.saos.persistence.model.SourceCode;
+import pl.edu.icm.saos.persistence.repository.JudgmentRepository;
+
+/**
+ * Finder of common court judgments that was removed
+ * from source external repository but still are present in the
+ * saos repository.
+ * 
```

Comment:
Finder of common court judgments that have been removed from the external source repository but are still present in the saos repository.